modules/lfs/pointer_scanner_nogogit.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+// +build !gogit
+
+package lfs
+
+import (
+	"code.gitea.io/gitea/modules/git"
+)
+
+// SearchPointerFiles not implemented
+func SearchPointerFiles(repo *git.Repository) ([]*Pointer, error) {
+	return []*Pointer{}, nil
```

Comment:
Yes it very much already is implemented.

 https://github.com/go-gitea/gitea/blob/81467e6f35f343b911c09f746deca869a48da4c8/routers/repo/lfs.go#L411

Reply:
Although very poorly commented there. (Sorry)

Read pipelines in reverse and compare with https://github.com/go-gitea/gitea/blob/fe79b13ab2ce9526fae10a5257b4e171cae30ecb/services/pull/lfs.go#L21

Reply:
Thank you, I didn't recognize that code. I moved the implementation into to `pointer_scanner_nogogit.go` file to unify the call site. The same could be done with the `LFSPush` method if `SearchPointerFiles` could handle "git ranges" instead of scanning the whole repository. But that should be another PR.